.github/scripts/validatePatchDetails.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+set -e
+
+ROOT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")/../.." && pwd)"
+DETAILS_MD="details.md"
+ERRORS=()
+
+# Get list of modified patch files in PR (compared to main branch)
+CHANGED_PATCHES=$(git diff --name-only origin/main | grep "^patches/.*\.patch$" || true)
```

Comment:
Probably want to add `--diff-filter=AM` here so you don't get an error about missing details when you _remove_ a patch